psl/psl/tests/functions/server_side_functions.rs

```diff
@@ -39,6 +39,25 @@ fn correctly_handle_server_side_cuid_function() {
         .assert_default_value(DefaultValue::new_expression(ValueGenerator::new_cuid()));
 }
 
+#[test]
```

Comment:
Same thing here, I'd just remove this test because it doesn't add anything really to help us. And we will delete all of these tests when we remove the dml module.